RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -132,7 +143,19 @@
 trackingPhase1QuadProp.toModify(mixedTripletStepSeedLayersB, layerList = ['BPix3+BPix4+TIB1'])
 
 # TrackingRegion
-mixedTripletStepTrackingRegionsB = mixedTripletStepTrackingRegionsA.clone(RegionPSet = dict(ptMin=0.6, originHalfLength=10.0))
+_mixedTripletStepTrackingRegionsB = mixedTripletStepTrackingRegionsA.clone(RegionPSet = dict(ptMin=0.6))
+mixedTripletStepTrackingRegionsB = _mixedTripletStepTrackingRegionsB.clone(RegionPSet = dict(originHalfLength=cms.double(10.0)))
+pp_on_XeXe_2017.toReplaceWith(mixedTripletStepTrackingRegionsB, _mixedTripletStepTrackingRegionsB)
```

Comment:
I think I understood the problem with my earlier proposal: Because earlier you replace the type of `mixedTripletStepTrackingRegionsA` from `GlobalTrackingRegionProducerFromBeamSpot` to `GlobalTrackingRegionWithVerticesProducer` (the former has this parameter but latter doesn't), and then `mixedTripletStepTrackingRegionsB` is cloned from ``mixedTripletStepTrackingRegionsA`.

I'd still like to get rid of the explicit type for `originHalfLength` though. Could you cherry-pick the commit https://github.com/makortel/cmssw/commit/b36d81ae97ed3312f1027d8aaac4a36722ba5c83, which I believe is the simplest way to achieve that?